boards/arm/samv7/same70-qmtech/include/board.h

```diff
@@ -46,11 +46,11 @@
  *   CPU clock: 120MHz
  *
  * There can be two on-board crystals.  However, the 32.768 crystal is not
- * populated on the stock SAME70.  The fallback is to use th on-chip, slow RC
- * oscillator which has a frequency of 22-42 KHz, nominally 32 KHz.
+ * populated on the stock SAME70.  The fallback is to use thee on-chip, slow
+ * RC oscillator which has a frequency of 22-42 KHz, nominally 32 KHz.
  */
 
-#undef  BOARD_HAVE_SLOWXTAL                   /* Slow crystal not populated */
+#define BOARD_HAVE_SLOWXTAL        1          /* Slow crystal is populated */
 #define BOARD_SLOWCLK_FREQUENCY    (32000)    /* 32 KHz RC oscillator (nominal)  */
```

Comment:
Shouldn't the slowclk frequency be updated to 32768 ? The Y1 crystal in the board is 32.768 KHz, I think 32000 is when using the slow internal RC

Reply:
Yes. You are right!